js/ui/text_box/ui.text_editor.base.js

```diff
@@ -74,6 +75,7 @@ const TextEditorBase = Editor.inherit({
 
         this._$beforeButtonsContainer = null;
         this._$afterButtonsContainer = null;
+        this._$labelContainerElement = null;
```

Comment:
```suggestion
        this._labelContainerElement = null;
```
The `$` prefix should indicate that the variable stores an instance of the jQuery (or Renderer) object.